apps/studio/schemas/timeRange.tsx

```diff
@@ -0,0 +1,88 @@
+import type { DateTimeInputProps } from "sanity";
+import { defineField, defineType, set, unset } from "sanity";
+import ct from "countries-and-timezones";
+
+const timezones = ct.getAllTimezones();
+
+const CustomDateTimeInput = ({ onChange, value }: DateTimeInputProps) => {
+  return (
+    <input
+      type="datetime-local"
+      onChange={(event) => {
+        console.log(
+          event.currentTarget.value,
+          // This is probably pretty fragile, but seems to work well enough?
+          // Basically appending "+00:00" to instruct the Date contructor to interpret our string as UTC.
+          // The other option would be parsing the string and using Date.UTC().
+          // idk time zones are hard :/
+          new Date(event.currentTarget.value + "+00:00")
+        );
+        onChange(
+          event.currentTarget.value
+            ? set(new Date(event.currentTarget.value + "+00:00").toISOString())
+            : unset()
+        );
+      }}
+      // Ensure pattern matches on browers that don't support `datetime-local`.
+      pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}"
+      value={value ? new Date(value).toISOString().slice(0, 16) : undefined}
+      style={{
+        borderRadius: "0.0625rem",
+        border: 0,
+        width: "100%",
+        boxSizing: "border-box",
+        outline: "none",
+        margin: 0,
+        backgroundColor: "#fff",
+        color: "#101112",
+        boxShadow: "inset 0 0 0 1px #ced2d9",
+        fontSize: "1rem",
+        lineHeight: "1.3125rem",
+        padding: "0.4375rem 0.75rem",
+      }}
+    />
+  );
+};
+
+export default defineType({
+  name: "timeRange",
+  title: "Time Range",
+  type: "object",
+  fields: [
+    defineField({
+      name: "timezone",
+      title: "Timezone",
+      type: "string",
+      initialValue: "America/Los_Angeles",
+      options: {
+        list: Object.entries(timezones).map(([key, timezone]) => ({
+          title: `${timezone.name} (${timezone.utcOffsetStr})`,
+          value: key,
+        })),
+      },
+      validation: (Rule) => Rule.required(),
+    }),
+    // The default behavior for datetime fields is to display / set the datetime based on the user's local timezone and store in UTC. For example if I'm in NY and select 9AM it'll be stored as 13:00 (assuming we're not in daylight savings), but still displayed as 9AM. This is neither configurable nor transparent to the user.
+    // I was unable to find a way to set the timezone of the datetime field, or disable the timezone conversion.
+    // Since there's no way to set the timezone, we need to be able to select the datetime as it's stored, hence the custom component. I've opted to use a native input field.
+    // I'm honestly quite shocked they have no way to take timezones into account.
```

Comment:
This is probably the best way to go. Just make sure to remove the comments on lines 66-68 since this is code can be viewable by anyone.

Reply:
I'd like to keep this comment to explain why I opted to use a custom component. Are you suggesting I remove it entirety, or just tone it down?

Reply:
Ah ok, in that case, just try to make it as formal as possible.